adapter/stdLogger/BUILD

```diff
@@ -0,0 +1,29 @@
+package(default_visibility = ["//visibility:public"])
+
+load("@io_bazel_rules_go//go:def.bzl", "go_library", "go_test")
+
+go_library(
+    name = "go_default_library",
+    srcs = [
+        "stdLogger.go",
```

Comment:
Could we go with stdioLogger as the name for this thing? I read "stdLogger" as "the standard logger", making everything else non-standard...